apartment_application_service/settings.py

```diff
@@ -176,8 +176,35 @@
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
-    "handlers": {"console": {"class": "logging.StreamHandler"}},
-    "loggers": {"django": {"handlers": ["console"], "level": env.str("LOG_LEVEL")}},
+    "formatters": {
+        "standard": {
+            "format": "%(asctime)s p%(process)d %(name)s %(levelname)s: %(message)s",
+            "datefmt": "%Y-%m-%dT%H:%M:%S.%03dZ",
+        }
+    },
+    "handlers": {
+        "console": {
+            "level": "INFO",
```

Comment:
This shouldn't be needed here as the logging configurations coming from DEFAULT_LOGGING settings https://github.com/django/django/blob/main/django/utils/log.py#L17